test/integration_test/basic_test.go

```diff
@@ -292,6 +300,11 @@ func BasicCsiRegression(tc *types.TestCase) func(*testing.T) {
 		// Create and validate StorageCluster
 		cluster = ci_utils.DeployAndValidateStorageCluster(cluster, ci_utils.PxSpecImages, t)
 
+		// Validate CSI is enabled by default
+		if cluster.Spec.FeatureGates["CSI"] != "true" {
+			require.NoError(t, fmt.Errorf("failed to validate CSI is enabled by default, it shouldn't be enabled by default, but it is set to %s", cluster.Spec.FeatureGates["CSI"]))
```

Comment:
can we just use `require.Equal(t, cluster.Spec.FeatureGates["CSI"], "true" )`?

Reply:
Added the check in the format @pure-jliao suggested to check if its enabled by default